examples/bipedal_walk.py

```diff
@@ -48,6 +48,7 @@
 
     # Added the callback functions
     print('*** SOLVE ' + key + ' ***')
+    sys.stdout.flush()
```

Comment:
We think we should remove all the `sys.stdout.flush()` in the example scripts. The reason is we just need them in the callback function.

Reply:
I won't mark all the cases as they are easy to spot.